packages/styles/src/themes/bootstrap/_overrides-color.scss

```diff
@@ -24,7 +24,8 @@ $light: color.$gray-background-light;
 $dark: $primary;
 
 // Colors that can be used as a background
-$theme-colors: color.$background-colors !default;
+$theme-colors: () !default;
+$colors: () !default;
```

Comment:
This kills every bootstrap color-affected utility and maybe some more components like cards, etc.:
- [background](https://getbootstrap.com/docs/5.2/utilities/borders/#color) (this is ok, because we removed the utility as well)
- [border](https://getbootstrap.com/docs/5.2/utilities/borders/#color)
- [color](https://getbootstrap.com/docs/5.2/utilities/borders/#color)

If you really like to do so, you need to remove the bootstrap-utilities as well!
And we must define another breakingchange, because classes like `.border-primary` and `.text-primary` do not work anymore.

Reply:
Re-introduced minimal set of bootstrap colors